### PR TITLE
Refactor random seed

### PR DESCRIPTION
**Pull request by rithgroove**
Make the simulator can reproduce result based on random seed

### CODE REVIEW

File: lib/Simulation/Business.py [hunk 1]
```diff
@@ -13,18 +12,27 @@ def __init__(self, building, businessData) -> None:
...
 
         # TODO: change randint to a random that takes the result from a normal curve
-        self.startHour =  random.randint (min_start_hour, max_start_hour)
-        workHours =  random.randint (min_workhour, max_workhour)
+        if (min_start_hour == max_start_hour):
+            self.startHour = min_start_hour
+        else:
+            self.startHour =  rng.integers(min_start_hour, max_start_hour)
+            
+        workHours =  rng.integers(min_workhour)
```
Comment (fhtanaka) [inline lib/Simulation/Business.py]: change to
```
workHours = min_workhour
```

Comment (rithgroove) [reply lib/Simulation/Business.py]: done

File: lib/Simulation/Job.py [hunk 2]
```diff
@@ -14,18 +14,34 @@ class Job:
...
+        
+        
+        
+       
+        
+        
+        
         indexes = np.where(jobClass.workDays)[0]
         if len(indexes) < self.activityPerWeek+1:
-            # TODO: Add an issue to move "prints" to a logging framework.
```
Comment (fhtanaka) [inline lib/Simulation/Job.py:25]: Keep this TODO comment for us to remember what we should do in the future

**Review by fhtanaka — APPROVED** [review]

**Review by jair-pereira — APPROVED** [review]
Approved.

File: lib/Map/Map.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 import geopy.distance as distance
 from pathlib import Path
 import csv
-import random
+
```
Comment (fhtanaka) [inline lib/Map/Map.py:8]: The function getRandomBuilding() is still using the random library

**Pull request merged**